documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+  something important being discussed.
+- Any round with a suggested length of time is subject to
+  [shortcutting as described below](#shortcutting-a-round) or to extension at
+  the discretion of the participants.
+
+### Question round
+
+This round lasts for 3 business days after the discussion starts but can be
+extended if questions reveal significant gaps in the proposal that must be
+understood before the proposal can be revised in the revision step. These
+questions are different from objections in that they’re not in the form of "we
+cannot do this because x part of the proposal will not work". Rather, they are
+clarifying questions in the form of:
+
+- "I don’t understand y in the proposal, can you clarify what is meant by this?"
+- "I think a gap exists in the proposed solution here [clarify where]. Can you
+  fill in the details in this area?"
+
+In short, the goal of the question round is for everyone participating to fully
+understand the proposal. Participants are urged that many objections can be
+restated as questions. This helps promote good faith between participants and
+allows the author to clarify things they might have missed in the initial draft
+of the proposal. This does _not_ mean that known paramount objections should be
+reserved by participants. If you know that a proposed solution is impossible or
+illegal, please do raise this concern as soon as possible.
+
+Note that the author does not have to respond to every question. However, they
+should keep them in mind during the revision round.
+
+#### Reactions
+
+This round also invites participants to share reactions, positive or negative,
+separate from potential paramount objections and in particular is an excellent
+venue to highlight positive aspects of the proposal regardless of whether
+paramount objections might be raised later. You may also share non-paramount
+objections at this round. For example, “I find JavaScript harder to work with
+than Python” is an objection but not a paramount objection. What makes an
+objection “paramount” is clarified further in the
+[Objection round](#objection-round) below.
+
+Reactions are explicitly expected at this round because, similar to questions,
+they can help prompt further clarification from the author to be address in the
+revision round.
+
+#### Purpose
+
+Proposals should all include an explicit "purpose". Often these are informed by
+the needs of a particular project associated with the proposal or with a
+requested discussion as the result of a retrospective or other self-feedback
+mechanism. However, contributors may, at their discretion, raise proposals for
+discussion that do not have prior conversations. In these cases, the question
+round is also understood to include an evaluation and clarification of the
+purpose of the proposal. For regularly requested proposals, however, the purpose
+is taken to be accepted and should not be the focus
```

Comment:
Is this in the wrong place? It also ends without a complete sentence.

Reply:
It was missing a full-stop. Fixed in the next commit.